runtime/src/main/java/io/quarkiverse/fury/ClassicFurySerializer.java

```diff
@@ -0,0 +1,80 @@
+package io.quarkiverse.fury;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.lang.annotation.Annotation;
+import java.lang.reflect.Type;
+
+import jakarta.ws.rs.Consumes;
+import jakarta.ws.rs.Produces;
+import jakarta.ws.rs.WebApplicationException;
+import jakarta.ws.rs.core.MediaType;
+import jakarta.ws.rs.core.MultivaluedMap;
+import jakarta.ws.rs.ext.MessageBodyReader;
+import jakarta.ws.rs.ext.MessageBodyWriter;
+import jakarta.ws.rs.ext.Provider;
+
+import org.apache.fury.BaseFury;
+import org.apache.fury.Fury;
+import org.apache.fury.ThreadSafeFury;
+import org.apache.fury.io.FuryInputStream;
+import org.apache.fury.resolver.ClassResolver;
+
+import io.quarkus.arc.Arc;
+
+@Provider
+@Consumes({ "application/fury", "application/*+fury" })
+@Produces({ "application/fury", "application/*+fury" })
+public class ClassicFurySerializer implements MessageBodyReader<Object>, MessageBodyWriter<Object> {
+    private BaseFury fury;
```

Comment:
Ha, it shoudl work. Let me try.

Reply:
No, it does not work. Because resteasy will init the providers at static_init time where the bean container is  not running. We have to retrieve `BaseFury` at runtime. see 
- #29